bin/setup_new

```diff
@@ -135,36 +135,36 @@ exit 1
   }
 }
 
-verify_database_configuration () {
-  if [ ! -f config/database.yml ]
+verify_database_migrations () {
+  bundle exec rake db:abort_if_pending_migrations
+  if [ $? -eq 1 ]
   then
-    cp config/database.travis.yml config/database.yml
-    bundle exec rake db:create:all
-
+    echo 'Creating the database, loading the schema and initializing it with the seed
+    data'
+    bundle exec rake db:setup
     if [ $? -ne 0 ]
     then
-      echo 'Unable to create the databases for you. Please ensure your database.yml is configured for your system and run rake db:create:all'
+      echo 'Unable to setup the databases for you. Please ensure your
+      database.yml is configured for your system and run rake db:setup'
       exit 0
     fi
   fi
 }
 
-verify_database_migrations () {
-  verify_database_configuration
-
-  bundle exec rake db:abort_if_pending_migrations
-  if [ $? -eq 1 ]
-  then
-    echo 'Running migrations...'
-    bundle exec rake db:setup db:test:prepare
```

Comment:
Removing db:test:prepare as it is [deprecated in Rails 4.1](https://github.com/rails/rails/pull/13528). (The test schema is maintained automatically).